modules/library/src/main/scala/zio/elasticsearch/query/sort/Sort.scala

```diff
@@ -26,6 +26,73 @@ sealed trait Sort {
   def paramsToJson: Json
 }
 
+object Sort {
+
+  /**
+   * Constructs an instance of [[SortByFieldOptions]] to sort by `_count` field in the context of an aggregation.
```

Comment:
Can any `Sort` be used in the context of aggregations or only these newly added ones?
Should we have an abstraction/restriction to distinguish sorting parameters for aggregation buckets?